### DISCUSSION TITLE
Wireframe

### DISCUSSION THREAD

**Discussion by ltuffery** [ltuffery/Matcha#44] [Ideas]
![image](https://github.com/user-attachments/assets/fe5d76d7-fe78-43c9-b3e2-6b267c1a0e32)

**Comment by ltuffery** [comment]
User page
![image](https://github.com/user-attachments/assets/084114ea-ec6f-4b1d-b330-02311ab8cd1b)

**Comment by swotex** [comment]
![boo](https://github.com/user-attachments/assets/b7cc2ff4-f90e-4f85-a231-c96857be5545)
